app/commands.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+import click
+from flask import Blueprint
+
+from .database import CatalogDBInterface
+from .models import Dataset
+from .opensearch import OpenSearchInterface
+
+search = Blueprint("search", __name__)
+
+
+@search.cli.command("sync")
+def sync_opensearch():
+    """Sync the datasets to the OpenSearch system."""
+
+    opensearch_host = os.getenv("OPENSEARCH_HOST")
+    if opensearch_host.endswith("es.awamzonaws.com"):
+        client = OpenSearchInterface(aws_host=opensearch_host)
+    else:
+        client = OpenSearchInterface(test_host=opensearch_host)
+
+    interface = CatalogDBInterface()
+    succeeded, failed = client.index_datasets(interface.db.query(Dataset))
```

Comment:
Since we're only running the materialized view daily, if this runs once a day we could be 1.99 days out of date on the opensearch instance. We should make sure to time this appropriately or chain these together to make sure we don't get too far behind...